### PR TITLE
Enhancement in Add Broadcast Page of Admin

### PR DESCRIPTION
**Pull request by sakshikhachane**
## Issue that this pull request solves

 Closes: #371 
## Proposed changes

- [x] Make the Page responsive
- [x]  Put everything on a card and add a submit button at the end of the card
- [x] Make sure to keep the navigation panel of the admin stick to left and render the edit toolbar on the right panel only

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot (197)
Screenshot (198)

### CODE REVIEW

**Comment by sakshikhachane** [comment]
@tejan-singh I had discussed this point - `On hover, the placeholder should appear with appropriate size.` with Kajol and she said to leave this point and make PR for the rest of the points. 
Please review the PR.

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: frontend/src/components/admin/add_broadcasts/add_broadcasts.js [hunk 1]
```diff
@@ -1,26 +1,141 @@
...
+                    Enter the Broadcast content here...
+                  </p>
+                  <SunEditor
+                    lang="en"
+                    placeholder="Please type here..."
+                    height="500px"
+                    className={styles.edit}
+                  />
+                  <div className={styles.submitBtn}>
+                    <Button2
```
Comment (tejan-singh) [inline frontend/src/components/admin/add_broadcasts/add_broadcasts.js:116]: - Please add `margin-top: 20px` on submit button

**Review by tejan-singh — CHANGES_REQUESTED** [review]

**Review by tejan-singh — CHANGES_REQUESTED** [review]
- Please remove all hover effects.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@sakshikhachane apart from what @tejan-singh suggested , just one more enhancement, Please put this folder `add_broadcasts` under `admin/broadcast` folder.

**Comment by sakshikhachane** [comment]
@tejan-singh I have done all the changes required. Please review the PR now.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**